ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2019.4.18f1
-m_EditorVersionWithRevision: 2019.4.18f1 (3310a4d4f880)
+m_EditorVersion: 2021.3.3f1
```

Comment:
I guess we will use the new version in the future

Reply:
Right. `2021.3.3f1` is current active LTS. We will update the new version when the next LTS is active.